mods/ts/sequences/infantry.yaml

```diff
@@ -530,7 +529,6 @@ umagon:
 	die6: electro
 		Length: *
 		Tick: 80
-	icon: umagicon
```

Comment:
Umagon was not buildable in the original, but I agree that removing this is unnecessary.